app/models/bartender.rb

```diff
@@ -1,5 +1,4 @@
 class Bartender < ApplicationRecord
   has_many :parties, dependent: :destroy
   has_many :users, through: :parties
-  has_many :reviews, dependent: :destroy
```

Comment:
Did you test this in the console? I think you have to say has_many :reviews, through: :parties, dependent: :destroy